### PR TITLE
Screen cams

### PR DESCRIPTION
**Pull request by dwantula**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,2 @@
+REACT_APP_API_URL=https://api.windy.com/api/webcams/v2/
+REACT_APP_API_KEY=kfAgnyEXUFixcGSQOVOro4fe4QP7XIO6
```
Comment (mi-chal) [inline .env]: You should not commit `.env` file

File: src/screens/Cams/Cams.jsx [hunk 2]
```diff
@@ -1,9 +1,73 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { fetchCountryList, fetchCityList } from 'shared/services/cams';
+
```
Comment (mi-chal) [inline src/screens/Cams/Cams.jsx:2]: No need for empty line

File: src/shared/components/Select/Select.jsx [hunk 3]
```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './styles.scss';
+
+const Select = ({ name, options, value, onChange, placeholder }) => {
+  return options ? (
```
Comment (mi-chal) [inline src/shared/components/Select/Select.jsx]: I don't think you need this ternary.

File: src/shared/services/cams.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
+import apiClient from 'shared/services/apiClient';
+
+export async function fetchCamsList() {
+  const response = await apiClient.get(
+    'list/country=PL/category=city/orderby=popularity/limit=20?show=webcams:location,image',
+  );
+  return response.data.result.webcams;
+}
+
+function sort(array, key) {
```
Comment (mi-chal) [inline src/shared/services/cams.js]: Move this function to `utills/sort.js`

File: src/shared/services/cams.js [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+    if (x > y) {
+      return 1;
+    }
+    return 0;
+  });
+}
+
+export async function fetchCountryList() {
+  const response = await apiClient.get('list?show=countries');
+  const countries = response.data.result.countries;
```
Comment (mi-chal) [inline src/shared/services/cams.js]: You could use destructure here

File: .env (hunk 1 above)
Comment (dwantula) [reply .env]: Done

File: src/screens/Cams/Cams.jsx (hunk 2 above)
Comment (dwantula) [reply src/screens/Cams/Cams.jsx:2]: Fixed

File: src/shared/components/Select/Select.jsx (hunk 3 above)
Comment (dwantula) [reply src/shared/components/Select/Select.jsx]: Fixed

File: src/shared/services/cams.js (hunk 4 above)
Comment (dwantula) [reply src/shared/services/cams.js]: Done

File: src/shared/services/cams.js (hunk 5 above)
Comment (dwantula) [reply src/shared/services/cams.js]: Done

**Pull request merged**